### DISCUSSION TITLE
Searchindex seems to be empty

### DISCUSSION THREAD

**Discussion by leganz** [statamic-rad-pack/meilisearch#11] [Help] [answered]
I was wondering if I am doing something wrong here.

my config looks like this => 

```php
'statamic_all' => [
            'driver' => 'meilisearch',
            'searchables' => ['collection:articles'],
            'fields' => ['id', 'title', 'url'],
            'settings' => [
                'filterableAttributes' => ['type'],
                'distinctAttribute' => 'thread',
                'stopWords' => ['der', 'die', 'das'],
                'sortableAttributes' => ['timestamp'],
                'rankingRules' => [
                    'sort',
                    'words',
                    'typo',
                    'proximity',
                    'attribute',
                    'exactness',
                ],
            ],
        ],
```

But after using the search index update command it seems that there are entries but they or empty? The search via api does not return any entries...

<img width="914" alt="CleanShot 2023-01-11 at 06 19 59@2x" src="https://user-images.githubusercontent.com/3373530/211723884-8197b11e-9d0e-406d-ba51-510145049c6f.png">

Does anybody have an idea why this might not work?

**Comment by tao** [comment, marked as answer]
Does your collection have a field called `thread` for grouping articles or was that copied over from the example by mistake? It might be trying to return entries with the distinct field that doesn't exist so it doesn't return anything 🤔

**Reply by leganz** [reply]
oh boy thanks! that was indeed a mistake! thanks for helping!
